playbook-website/app/views/guides/getting_started/swift_setup.md

```diff
@@ -30,24 +29,15 @@ Built and maintained by the User Experience Team at [Power Home Remodeling](http
     ```
 5. Build the project
 
-### Install [`pre-commit`](https://pre-commit.com/#install) via `brew install pre-commit`
-  1. In the Playbook Swift root directory, run `pre-commit install` to set up the git hook scripts
+# Code Format
 
-# Code Format & Linting
-
-See the [Editing Linting, Formatting, and Auto Correction](LINTING.md) doc for more information.
+Playbook Swift enforces an indentation width of 2 spaces.
 
 ## Component Examples via PlaybookShowcase
 
 Playbook Swift contains the app named `PlaybookShowcase`; providing examples of each component. PlaybookShowcase can be launched within the XCode simulator, side-loaded, or downloaded to your device.
 
-# Note: YAML Configuration Files
-
-- You may need to add the package like so:
+You can find Playbook Swift documentation for a particular component following this example path in the playbook-website repo: playbook/app/pb_kits/playbook/pb_[COMPONENT_NAME]/docs. The relevant files will end with _swift.md.
 
-```yaml
-packages:
-  Playbook:
-    url: git@github.com:powerhome/PlaybookSwift.git
-    version x.x.x // Where x.x.x is the desired version
-```
\ No newline at end of file
+# Contribution Documentation
+Contribution documentation can be found [here](https://github.com/powerhome/playbook-swift/blob/main/Contribution.md).
```

Comment:
Once you've installed Playbook Swift into your project and kicked its tires a bit, you may find that you'd like to contribute to its growth and development to better-serve your needs (and all other Playbook users), and we'd love to hear from you! Please see our [contribution docs](https://github.com/powerhome/playbook-swift/blob/main/Contribution.md) for more information on how to help us continuing growing Playbook Swift.